### DISCUSSION TITLE
Flags & Icons in front-end theme

### DISCUSSION THREAD

**Discussion by AIC-BV** [wintercms/winter#318] [Q&A] [answered]
Hello,

I want to use font-awesome in my website. I noticed that WinterCMS has their own set based on font-awesome.

Is it possible to use the flags and icons in the front-end, do I have to use font-awesome or can I use WinterCMS icons?
[https://wintercms.com/docs/ui/flag](https://wintercms.com/docs/ui/flag)
[https://wintercms.com/docs/ui/icon](https://wintercms.com/docs/ui/icon)

I was hoping I could include the flags and icons css like this:
`<link href='{{ ["@framework.extras", "@flags", "@icons", "assets/css/style.css"] | theme }}' rel='stylesheet' type='text/css'>`

and then use it like so:
`<a href='https://www.facebook.com' target='_blank'><i class="icon-facebook"></i></a>`

**Comment by bennothommo** [comment, marked as answer]
We try to keep the assets of the Storm UI library separate from the CMS/front-end, so that you are not tied to using our front-end frameworks for your projects.

I would suggest in this instance that you use Font Awesome directly.

**Reply by LukeTowers** [reply]
I agree with @bennothommo on this, if we were to provide those assets in the manner you're suggesting then it makes it even more difficult to make any changes to them in the future because now any potential breakages wouldn't be contained to just the backend but also to the frontend, which is a huge risk for us and our users to take on and not one that we would want to encourage.

**Reply by AIC-BV** [reply]
Ok, that sounds obvious!
Thanks!
